customization/base_workflow.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Base Workflow
+parent: Customization
+nav_order: 1
+---
+
+The `mappers` folder contains `base_workflow.osw` which serves as a simulation input for URBANopt. It is an OpenStudio workflow file that contains URBANopt Measures and dictates the sequence of running these Measures. Measures are added to create building models and apply different energy conservation Measures for different Scenarios.  
+
+The URBANopt GeoJSON Workflow uses building footprints to create building models.  The diagram below contains details on the sequence and purpose of measures used to accomplish this task.
+
+![urbanopt measure workflow diagram](../doc_files/urbanopt_measure_workflow.png)
+
+All measures contained in the base workflow are listed below:
+
+- [`set_run_period`](https://github.com/NREL/openstudio-common-measures-gem/tree/master/lib/measures/set_run_period): An OpenStudio Measure used to define the number of timesteps per hour and specify the begin and end date for running the simulation.
+
+- [`ChangeBuildingLocation`](https://github.com/NREL/openstudio-common-measures-gem/tree/master/lib/measures/ChangeBuildingLocation): An OpenStudio Measure used to specify and load the EPW file.
+
+- [`create_bar_from_building_type_ratios`](https://github.com/NREL/openstudio-model-articulation-gem/tree/master/lib/measures/create_bar_from_building_type_ratios): An OpenStudio Model Articulation Measure used to create a core and perimeter bar sliced by space type. It takes in one or more building types as user arguments to create space type collections.
+
+- [`create_typical_building_from_model 1`](https://github.com/NREL/openstudio-model-articulation-gem/tree/master/lib/measures/create_typical_building_from_model): An OpenStudio Model Articulation Measure that creates a custom prototype building with user-defined geometry and assigns constructions, schedules, internal loads, HVAC and other loads such as exterior lights and service water heating based on the space and sub space types. The `add_hvac` is set to `false` by default.
```

Comment:
Can we add a bit at the very end of this measure that tells people the `add_hvac` will get taken care of later? Perhaps: "The `add_hvac` is set to `false` by default in this measure because it gets handled later in the process to account for blended space types."